test/fixtures/incorrect.js

```diff
@@ -384,12 +384,27 @@ if (true === yoda) {
 // `react-hooks/rules-of-hooks`.
 const useEffect = noop;
 
-const ReactComponent = () => {
+const RulesOfHooks = () => {
   if (Math.random() > 0.5) {
     useEffect(noop);
   }
 
   return null;
 };
 
-noop(ReactComponent);
+noop(RulesOfHooks);
+
+// `react/jsx-tag-spacing`.
+const React = null;
+
+const TagSpacingBeforeClosing = () => (
```

Comment:
This is actually testing the `beforeSelfClosing` option, so please rename this to `TagSpacingBeforeSelfClosing`.

Also, please add tests for the `beforeClosing` and `closingSlash` options.